_maps/shuttles/whiteship_meta.dmm

```diff
@@ -3,38 +3,13 @@
 /turf/open/space,
 /area/space)
 "ab" = (
```

Comment:
Er, did you do mapmerge TGM or convert to TGM with this? Why is it rewriting all the keys?

Reply:
I originally commited the map without converting to TGM so it would basically be seen as a new file, then converted to TGM and commited again. I'm not seeing anything wrong in the file at a first glance though.

Reply:
probably has to do with changing the map size?